template/nuxt/pages/index.vue

```diff
@@ -28,7 +28,11 @@
   </div>
 </template>
 
+<%_ if (typescript) { _%>
+<script lang="ts">
```

Comment:
I would replace
```js
export default {
  components: {
    Logo
  }
}	
```
by
```js
import Vue from 'vue'

export default Vue.extend({
  components: {
    Logo
  }
})
```

See : https://typescript.nuxtjs.org/cookbook/components/

Furthermore, I'm wondering if we could output the documentation link after the project is generated, so people directly can jump into CookBook).

Reply:
thanks @kevinmarrec 

- replaced script tag for ts.
- added cookbook link when ts users